Rule add: windows access token abuse

YamatoSecurity opened the pull request:
Closes https://github.com/SigmaHQ/sigma/issues/2964

frack113 commented on rules/windows/builtin/security/win_security_access_token_abuse.yml:
```diff
@@ -0,0 +1,29 @@
+status: experimental
+description: 'This rule tries to detect token impersonation and theft. (Example: DuplicateToken(Ex) and ImpersonateLoggedOnUser with the LOGON32_LOGON_NEW_CREDENTIALS flag.)'
+references:
+    - https://attack.mitre.org/techniques/T1134/001/
```
Find these refs : 
 - https://www.elastic.co/fr/blog/how-attackers-abuse-access-token-manipulation 
 - https://www.manageengine.com/log-management/cyber-security/access-token-manipulation.html

YamatoSecurity replied:
Thank you for the references. I would be happy to add them.

nasbench commented:
A couple of things.

- The issue mentioned classifies this rule as `medium` with AV as False Positives and now it's `high`. Why is that?

- There is a rule closely similar to the logic of this - [192a0330-c20b-4356-90b6-7b7049ae0b87](https://github.com/SigmaHQ/sigma/blob/8b749fb1260b92b9170e4e69fa1bd2f34e94d766/rules/windows/builtin/security/win_security_overpass_the_hash.yml) I wonder if we can combine them to avoid duplicate alerts.

YamatoSecurity commented:
> * The issue mentioned classifies this rule as `medium` with AV as False Positives and now it's `high`. Why is that?

In my testing, I didn't receive any FPs so I raised it to `high`. If you are getting FPs on your datasets, please let me know. I'm fine with lowering it to `medium`.

> * There is a rule closely similar to the logic of this - [192a0330-c20b-4356-90b6-7b7049ae0b87](https://github.com/SigmaHQ/sigma/blob/8b749fb1260b92b9170e4e69fa1bd2f34e94d766/rules/windows/builtin/security/win_security_overpass_the_hash.yml) I wonder if we can combine them to avoid duplicate alerts.

Indeed, it might be checking the same thing. I am fine with combining rules to avoid duplicate alerts. This rule is rated as `high` as well.

YamatoSecurity commented:
I didn't have time to test out the differences of when the LogonProcess is `Advapi` and when it is `seclogo`. It is possible that overpass the hash results in a logon process from `seclogo` and general access token abuse is with `Advapi`. In that case, we can separate the rules according to LogonProcess.
By the way, in the original rule submission it said `ElevatedToken: 'No'`, however, with the datasets available to me, I have noticed that sometimes it is No and sometimes Yes depending on the user's privileges which is why I omitted it.

YamatoSecurity commented:
I took out `seclogo` to avoid overlap with [192a0330-c20b-4356-90b6-7b7049ae0b87](https://github.com/SigmaHQ/sigma/blob/8b749fb1260b92b9170e4e69fa1bd2f34e94d766/rules/windows/builtin/security/win_security_overpass_the_hash.yml), added Anti-Virus as possible FP, and changed the level to medium.
